common/src/test/java/org/broadleafcommerce/common/rule/MvelHelperTest.java

```diff
@@ -112,45 +112,9 @@ public void testRequestMapProperty() {
         boolean result = MvelHelper.evaluateRule("request.properties['blcSearchTerm'] == 'hot'", parameters);
         assertTrue(result);
     }
-
-    /**
-     * Confirms MVEL failure for special overloaded method case.
-     * </p>
-     * During compilation, if an mvel expression contains a method calls with params, mvel will attempt to identify a perfect
-     * method signature match based on the types of the params. However, if the method is overloaded and one or more of
-     * the params are null, mvel will not be able to 100% identify the right method version if the overloaded methods
-     * have the same quantity of params and have type overlap. Take this example:
-     * </p>
-     * {@code SelectizeCollectionUtils#intersection(String param, Iterable param2);}
-     * {@code SelectizeCollectionUtils#intersection(Iterable param, Iterable param2);}
-     * </p>
-     * If a rule is executed calling the intersection method and a null is passed in for the first parameter, mvel will
-     * not know which method to pick, so it will end up picking one of them. This has proven to be undetermined at runtime,
-     * which has made the problem even harder to identify. Furthermore, once the choice is made during compilation,
-     * that compiled expression will utilize the "incorrect" choice going forward, which will cause the expression to
-     * fail, even when neither param is null. In fact, in the case above, mvel will "coerce" the first param Iterable
-     * instance into a String (i.e. flatten the list representation into a comma delimited String) in order to continue
-     * calling the method identified during compilation. This causes the expression to permanently be in an incorrect
-     * state from which it will never recover.
-     */
-    public void testMvelMethodOverloadFailureCase() throws IOException {
```

Comment:
Why did you delete this test?

Reply:
this test failures while project building. I found that in 5.2 this test was deleted as not necessary because mvel version upgraded. I need to build framework version 5.0 because I need to upgrade woodstox library version, because old version causes error working with Apache CXF

Reply:
Context this task https://github.com/BroadleafCommerce/blc-cybersource-payment/pull/15